imitation/scripts/data_collect.py

```diff
@@ -18,15 +21,91 @@ def make_PPO2(env_name, num_vec, **make_blank_policy_kwargs):
 
 
 @data_collect_ex.main
-def main(env_name, total_timesteps, num_vec=8, make_blank_policy_kwargs={}):
+def main(env_name, total_timesteps,
+         *,
+         num_vec=8,
+         make_blank_policy_kwargs={},
+
+         rollout_save: bool = False,
+         rollout_save_interval: Optional[int] = None,
+         rollout_save_n_samples: Optional[int] = None,
+         rollout_dir: Optional[str] = None,
+
+         policy_save: bool = False,
+         policy_save_interval: Optional[int] = None,
+         policy_dir: Optional[str] = None,
+         ):
   tf.logging.set_verbosity(tf.logging.INFO)
 
   policy = make_PPO2(env_name, num_vec, **make_blank_policy_kwargs)
 
-  callback = util.make_save_policy_callback("data/")
+  callback = _make_callback(
+    rollout_save, rollout_save_interval, rollout_save_n_samples,
+    rollout_dir, policy_save, policy_save_interval, policy_dir)
   policy.learn(total_timesteps, callback=callback)
```

Comment:
It seems the only way to save a rollout or policy is via the callback. What if I don't want intermediate checkpoints, and just want to save the final policy? I think it'd be reasonable to always save the final policy and rollouts, and make checkpoints optional.

Related to this, I think the logic to save rollouts and the policy can be factored out into two helper functions, that is called from within `callback` but can also be called elsewhere.

Reply:
> always save the final policy and rollouts, and make checkpoints optional.
Agreed, this is better.

>  I think the logic to save rollouts and the policy can be factored out into two helper functions.
Seems reasonable, will look more closely into this